internal/pkg/externalplugins/label/label.go

```diff
@@ -59,15 +59,15 @@ func HelpProvider(epa *externalplugins.ConfigAgent) func(
 
 		pluginHelp := &pluginhelp.PluginHelp{
 			Description: "The label plugin provides commands that add or remove certain types of labels. " +
-				"Labels of the following types can be manipulated: 'status/*'," +
-				" 'sig/*', and 'type/*'. More labels can be configured to be used via the /label command.",
+				"For example, the labels of the following types: 'status/*'," +
```

Comment:
I don't think this is clearly described, we can label by type or use `/label` to label additional labels.

Reply:
done

Reply:
There is still a problem, you can refer to https://book.prow.tidb.io/plugins/label.html.